### PR TITLE
More compatibility updates

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: com.pilosa.client/src/main/java/com/pilosa/client/Internal.java [hunk 1]
```diff
@@ -399,9 +403,9 @@ public int hashCode() {
       }
       int hash = 41;
       hash = (19 * hash) + getDescriptorForType().hashCode();
-      if (getBitsCount() > 0) {
-        hash = (37 * hash) + BITS_FIELD_NUMBER;
-        hash = (53 * hash) + getBitsList().hashCode();
+        if (getColumnsCount() > 0) {
```
Comment (tgruben) [inline com.pilosa.client/src/main/java/com/pilosa/client/Internal.java]: I know this is a bit of pain but I feel the singular form (Column) is a bit more natural in this entire PR.

File: com.pilosa.client/src/main/java/com/pilosa/client/Internal.java [hunk 2]
```diff
@@ -5744,13 +5761,13 @@ public int getSerializedSize() {
...
       }
-      if (excludeAttrs_ != false) {
+        if (excludeRowAttrs_ != false) {
         size += com.google.protobuf.CodedOutputStream
-          .computeBoolSize(6, excludeAttrs_);
-      }
-      if (excludeBits_ != false) {
+                .computeBoolSize(6, excludeRowAttrs_);
+        }
+        if (excludeColumns_ != false) {
```
Comment (tgruben) [inline com.pilosa.client/src/main/java/com/pilosa/client/Internal.java]: though in this particular place the plural sounds more appropriate.

File: com.pilosa.client/src/main/java/com/pilosa/client/Internal.java (hunk 1 above)
Comment (yuce) [reply com.pilosa.client/src/main/java/com/pilosa/client/Internal.java]: `Internal.java` is automatically generated by protobuf compiler, but I'll look out for `getColumnsCount` in manual written code

File: com.pilosa.client/src/main/java/com/pilosa/client/Internal.java (hunk 2 above)
Comment (yuce) [reply com.pilosa.client/src/main/java/com/pilosa/client/Internal.java]: Likewise, automatically generated.

**Comment by yuce** [comment]
Great!

**Pull request merged**